### PR TITLE
Nyala 10 progress tracking

### PR DESCRIPTION
**Pull request by kosenchiha**
Save lecture for a logged in user as completed when user has watched 80% of the lecture.
If data was saved successfully, user see a toast message, which says that lecture is completed.

### CODE REVIEW

File: src/pages/signup/index.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ const Signup = ({
                 htmlFor="password"
                 className="font-bold text-grey-darker block mb-2"
               >
-                {t('email')}
+                {t('password')}
```
Comment (abolkog) [inline src/pages/signup/index.js:72]: nice catch :)

File: src/templates/LectureView/VideoPlayer.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    currentUser.profile &&
+    currentUser.profile.completedlectures
+      ? currentUser.profile.completedlectures
+      : {}
+
+  const updateCompletedLectures = async updatedLectures => {
+    try {
+      const response = await addProfile({ completedlectures: updatedLectures })
+      setCurrentUser({ ...currentUser, profile: response })
+      toast.success(t('lectureCompletedMsg'))
```
Comment (abolkog) [inline src/templates/LectureView/VideoPlayer.js:27]: Can we remove this toast message, please. I think later on we will do a UI change on for the lecture list to highlight the completed lectures

File: src/templates/LectureView/VideoPlayer.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <div className="relative" style={{ paddingTop: '56.25%' }}>
+      <ReactPlayer
+        className="absolute top-0 right-0"
+        url={url}
+        width="100%"
+        height="100%"
+        onProgress={isLoggedIn ? handleProgress : () => {}}
+      />
```
Comment (abolkog) [inline src/templates/LectureView/VideoPlayer.js:57]: Its not part of this PR/JIRA story but can we squeeze in one more prop her show the controls, props name is `controls` more info [here](https://www.npmjs.com/package/react-player#props)

```suggestion
      <ReactPlayer
        className="absolute top-0 right-0"
        url={url}
        width="100%"
        height="100%"
        onProgress={isLoggedIn ? handleProgress : () => {}}
        controls
      />
```

File: src/templates/LectureView/VideoPlayer.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      ? currentUser.profile.completedlectures
+      : {}
+
+  const updateCompletedLectures = async updatedLectures => {
+    try {
+      const response = await addProfile({ completedlectures: updatedLectures })
+      setCurrentUser({ ...currentUser, profile: response })
+      toast.success(t('lectureCompletedMsg'))
+    } catch (err) {
+      console.log(err)
```
Comment (abolkog) [inline src/templates/LectureView/VideoPlayer.js:29]: We do not have error handling for now, we can just add FIXME 
```suggestion
      // FIXME: error handling
      // console.log(err)
```

**Review by abolkog — CHANGES_REQUESTED** [review]
Nice work, all good. Just minor changes and ready to merge

File: src/templates/LectureView/VideoPlayer.js (hunk 2 above)
Comment (kosenchiha) [reply src/templates/LectureView/VideoPlayer.js:27]: Removed

File: src/templates/LectureView/VideoPlayer.js (hunk 3 above)
Comment (kosenchiha) [reply src/templates/LectureView/VideoPlayer.js:57]: done

File: src/templates/LectureView/VideoPlayer.js (hunk 4 above)
Comment (kosenchiha) [reply src/templates/LectureView/VideoPlayer.js]: All done

**Review by abolkog — APPROVED** [review]
